packages/contracts/deploy/helpers.ts

```diff
@@ -167,26 +176,71 @@ export async function checkSetManagingDao(
   }
 }
 
-export async function checkPermission(
-  permissionManager: Contract,
-  where: string,
-  who: string,
-  permission: string,
-  data = '0x'
-) {
+export interface CheckPermission {
+  isGrant: boolean;
+  permissionManager: ethers.Contract;
+  where: string;
+  who: string;
+  permission: string;
+  data?: string;
+}
+
+export async function checkPermission({
+  isGrant,
+  permissionManager,
+  where,
+  who,
+  permission,
+  data = '0x',
+}: CheckPermission) {
   const permissionId = ethers.utils.id(permission);
   const isGranted = await permissionManager.callStatic.isGranted(
```

Comment:
Not your written code, but no idea why we use `callStatic`.  `isGranted` is `view`.

Reply:
amended